lib/network_test.go

```diff
@@ -104,14 +105,76 @@ var expectedBlockHeader = &MsgDeSoHeader{
 	ExtraNonce: uint64(101234123456789),
 }
 
+// Creates fully formatted a PoS block header with random signatures
+// and block hashes
+func createTestBlockHeaderVersion2(t *testing.T) *MsgDeSoHeader {
+	testBlockHash := BlockHash{
+		0x00, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x10, 0x11,
+		0x12, 0x13, 0x14, 0x15, 0x16, 0x17, 0x18, 0x19, 0x20, 0x21,
+		0x22, 0x23, 0x24, 0x25, 0x26, 0x27, 0x28, 0x29, 0x30, 0x31,
+		0x32, 0x33,
+	}
+	testMerkleRoot := BlockHash{
+		0x00, 0x35, 0x36, 0x37, 0x38, 0x39, 0x40, 0x41, 0x42, 0x43,
+		0x44, 0x45, 0x46, 0x47, 0x48, 0x49, 0x50, 0x51, 0x52, 0x53,
+		0x54, 0x55, 0x56, 0x57, 0x58, 0x59, 0x60, 0x61, 0x62, 0x63,
+		0x64, 0x65,
+	}
+
+	_, testBLSSignature := _generateValidatorVotingPublicKeyAndSignature(t)
```

Comment:
can we define this somewhere such that postgres tests compile? we don't run this test in postgres CI so the function can simply panic.

Reply:
Sounds good. I had to create a new file for it with a `!relic` tag. LMK if you want me to place the file anywhere specific

Reply:
I like how you handled it - good stuff.